java/src/main/java/com/powsybl/python/network/NetworkCFunctions.java

```diff
@@ -132,8 +142,13 @@ public static ObjectHandle loadNetwork(IsolateThread thread, CCharPointer file,
         return doCatch(exceptionHandlerPtr, () -> {
             String fileStr = CTypeUtil.toString(file);
             Properties parameters = createParameters(parameterNamesPtrPtr, parameterNamesCount, parameterValuesPtrPtr, parameterValuesCount);
-            Reporter reporter = ReportCUtils.getReporter(reporterHandle);
-            Network network = Network.read(Paths.get(fileStr), CommonObjects.getComputationManager(), ImportConfig.load(), parameters, IMPORTERS_LOADER_SUPPLIER.get(), reporter);
+            ReporterModel reporter = ObjectHandles.getGlobal().get(reporterHandle);
+            Network network;
+            if (reporter == null) {
+                network = Network.read(Paths.get(fileStr), LocalComputationManager.getDefault(), ImportConfig.load(), parameters);
+            } else {
+                network = Network.read(Paths.get(fileStr), LocalComputationManager.getDefault(), ImportConfig.load(), parameters, IMPORTERS_LOADER_SUPPLIER.get(), reporter);
```

Comment:
I would recommand to use one method (the more complex) and just adapt Reporter (a real one or NO_OP one for better clarity.